frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,9 @@
 import React from 'react'
-//Import components that you create here
+import Tools from '../Tools'
```

Comment:
When I run the app it says that it cant resolve this component. I think if you move the Tools component to the LandingPage folder and import the file it will work.

Reply:
I believe to fix this problem you'll need to add another /Tools at the end so instead of looking at the Tools folder, the program will find the specific Tools.js file. Also I think you'll need to change the first part of the import to just one dot, so it looks for the Tools folder in the current directory (assuming you have moved the Tools folder into the LandingPage folder).

Reply:
Yeah, I thought I updated the path after moving some things around but it looks like I forgot. Fixed now though.